docker/app/migrate_and_serve.sh

```diff
@@ -9,7 +9,12 @@
 
 echo [i] Applying migrations.
 python manage.py migrate --verbosity 1
+
 echo [i] Collecting static files.
 python manage.py collectstatic --no-input --clear --verbosity 0
+
+echo [i] Creating a superuser
+echo "from django.contrib.auth import get_user_model; User = get_user_model(); User.objects.create_superuser('admin', 'admin', 'admin')" | python manage.py shell
```

Comment:
Unfortunately, this will fail the second time you start the docker-compose, since the username already exists at that point. See trace back at the bottom.

This solves that and prints "Account already exists" when it already exists:

```suggestion
echo "from django.contrib.auth import get_user_model; User = get_user_model(); User.objects.create_superuser('admin', 'admin', 'admin') if not User.objects.filter(username='admin').exists() else print('Username already exists')" | python manage.py shell
```

The full traceback you'll get otherwise:

```py
web_1       | [i] Creating a superuser
postgres_1  | 2019-04-18 13:23:51.787 UTC [26] ERROR:  duplicate key value violates unique constraint "auth_user_username_key"
postgres_1  | 2019-04-18 13:23:51.787 UTC [26] DETAIL:  Key (username)=(admin) already exists.
postgres_1  | 2019-04-18 13:23:51.787 UTC [26] STATEMENT:  INSERT INTO "auth_user" ("password", "last_login", "is_superuser", "username", "first_name", "last_name", "email", "is_staff", "is_active", "date_joined") VALUES ('pbkdf2_sha256$150000$FoXDVt2Gbs85$gpB7NgN9RD7Z+R/AVOXawJW0Yoq1SoTXvSjB0s0AWnc=', NULL, true, 'admin', '', '', 'admin', true, true, '2019-04-18T13:23:51.591734+00:00'::timestamptz) RETURNING "auth_user"."id"
web_1       | Traceback (most recent call last):
web_1       |   File "/opt/bitnami/python/lib/python3.7/site-packages/django/db/backends/utils.py", line 84, in _execute
web_1       |     return self.cursor.execute(sql, params)
web_1       | psycopg2.errors.UniqueViolation: duplicate key value violates unique constraint "auth_user_username_key"
web_1       | DETAIL:  Key (username)=(admin) already exists.
web_1       | 
web_1       | 
web_1       | The above exception was the direct cause of the following exception:
web_1       | 
web_1       | Traceback (most recent call last):
web_1       |   File "manage.py", line 21, in <module>
web_1       |     main()
web_1       |   File "manage.py", line 17, in main
web_1       |     execute_from_command_line(sys.argv)
web_1       |   File "/opt/bitnami/python/lib/python3.7/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
web_1       |     utility.execute()
web_1       |   File "/opt/bitnami/python/lib/python3.7/site-packages/django/core/management/__init__.py", line 375, in execute
web_1       |     self.fetch_command(subcommand).run_from_argv(self.argv)
web_1       |   File "/opt/bitnami/python/lib/python3.7/site-packages/django/core/management/base.py", line 323, in run_from_argv
web_1       |     self.execute(*args, **cmd_options)
web_1       |   File "/opt/bitnami/python/lib/python3.7/site-packages/django/core/management/base.py", line 364, in execute
web_1       |     output = self.handle(*args, **options)
web_1       |   File "/opt/bitnami/python/lib/python3.7/site-packages/django/core/management/commands/shell.py", line 92, in handle
web_1       |     exec(sys.stdin.read())
web_1       |   File "<string>", line 1, in <module>
web_1       |   File "/opt/bitnami/python/lib/python3.7/site-packages/django/contrib/auth/models.py", line 162, in create_superuser
web_1       |     return self._create_user(username, email, password, **extra_fields)
web_1       |   File "/opt/bitnami/python/lib/python3.7/site-packages/django/contrib/auth/models.py", line 145, in _create_user
web_1       |     user.save(using=self._db)
web_1       |   File "/opt/bitnami/python/lib/python3.7/site-packages/django/contrib/auth/base_user.py", line 66, in save
web_1       |     super().save(*args, **kwargs)
web_1       |   File "/opt/bitnami/python/lib/python3.7/site-packages/django/db/models/base.py", line 741, in save
web_1       |     force_update=force_update, update_fields=update_fields)
web_1       |   File "/opt/bitnami/python/lib/python3.7/site-packages/django/db/models/base.py", line 779, in save_base
web_1       |     force_update, using, update_fields,
web_1       |   File "/opt/bitnami/python/lib/python3.7/site-packages/django/db/models/base.py", line 870, in _save_table
web_1       |     result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
web_1       |   File "/opt/bitnami/python/lib/python3.7/site-packages/django/db/models/base.py", line 908, in _do_insert
web_1       |     using=using, raw=raw)
web_1       |   File "/opt/bitnami/python/lib/python3.7/site-packages/django/db/models/manager.py", line 82, in manager_method
web_1       |     return getattr(self.get_queryset(), name)(*args, **kwargs)
web_1       |   File "/opt/bitnami/python/lib/python3.7/site-packages/django/db/models/query.py", line 1186, in _insert
web_1       |     return query.get_compiler(using=using).execute_sql(return_id)
web_1       |   File "/opt/bitnami/python/lib/python3.7/site-packages/django/db/models/sql/compiler.py", line 1332, in execute_sql
web_1       |     cursor.execute(sql, params)
web_1       |   File "/opt/bitnami/python/lib/python3.7/site-packages/django/db/backends/utils.py", line 99, in execute
web_1       |     return super().execute(sql, params)
web_1       |   File "/opt/bitnami/python/lib/python3.7/site-packages/django/db/backends/utils.py", line 67, in execute
web_1       |     return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
web_1       |   File "/opt/bitnami/python/lib/python3.7/site-packages/django/db/backends/utils.py", line 76, in _execute_with_wrappers
web_1       |     return executor(sql, params, many, context)
web_1       |   File "/opt/bitnami/python/lib/python3.7/site-packages/django/db/backends/utils.py", line 84, in _execute
web_1       |     return self.cursor.execute(sql, params)
web_1       |   File "/opt/bitnami/python/lib/python3.7/site-packages/django/db/utils.py", line 89, in __exit__
web_1       |     raise dj_exc_value.with_traceback(traceback) from exc_value
web_1       |   File "/opt/bitnami/python/lib/python3.7/site-packages/django/db/backends/utils.py", line 84, in _execute
web_1       |     return self.cursor.execute(sql, params)
web_1       | django.db.utils.IntegrityError: duplicate key value violates unique constraint "auth_user_username_key"
web_1       | DETAIL:  Key (username)=(admin) already exists.
```